File: Java/1002. Find Common Characters.java
class Solution {
    public List<String> commonChars(String[] words) {
        List<String> res = new ArrayList<>();
        for(int i=0;i<words[0].length();i++){
            String c=words[0].charAt(i)+"";
            int flag = 1;
            for(int j = 1; j < words.length; j++){
                if(words[j].contains(c)){
                    words[j]=words[j].replaceFirst(c,"");    
                }
                else{
                    flag = 0;
                    break;
                }
            }
            if(flag!=0){
                res.add(c);
            }
        }
        return res;
    }
}

File: C#/1002. Find Common Characters.cs
public class Solution {
    public IList<string> CommonChars(string[] words) {
        List<string> res = new List<string>();
        foreach(char c in words[0]){
            int flag = 1;
            for(int i = 1; i < words.Length; i++){
                if(words[i].Contains(c)){
                    words[i] = words[i].Remove(words[i].IndexOf(c), 1); 
                }
                else{
                    flag = 0;
                    break;
                }
            }
            if(flag!=0){
                res.Add(c.ToString());
            }
        }
        return res;
    }
}
